docs/resources/emoji.mdx

```diff
@@ -30,6 +30,12 @@ An emoji cannot have both subscription roles and non-subscription roles.
 Emojis with subscription roles are considered premium emoji, and count toward a separate limit of 25.  
 Emojis cannot be converted between normal and premium after creation.
 
+###### Emoji Formats
+
+Emojis can be uploaded as JPEG, PNG, GIF, WebP, and AVIF formats. All emojis (regardless of original format) can be served as WebP. We highly recommend that developers request emojis as WebP for maximum performance and compatibility. The Discord client uses WebP for all emojis displayed in-app.
```

Comment:
Rather than "request" emoji, should it be "submit emoji" (across the board)?

"request" feels like you are asking someone to submit something to me. Unless I'm missing a chunk of context, which is possible.

Reply:
I think that is referring to fetching the images that already exist, not uploading new emoji

Reply:
Maybe a silly question, since I don't know this area as well as others -- how are you requesting the emoji? Is there an API for requesting an emoji from somewhere? (that's what it reads like to me).

If not, I'd lean more towards something like:

```suggestion
Emojis can be uploaded as JPEG, PNG, GIF, WebP, and AVIF formats. All emojis (regardless of original format) can be served as WebP. We highly recommend that developers choose WebP emoji for maximum performance and compatibility. The Discord client uses WebP for all emojis displayed in-app.
```

Since that clears any ambiguity. WDYT?

Reply:
You're right that the wording is ambiguous. 'Request' here refers to when developers fetch emoji (`GET` requests), not when they upload them.  I'll update the text to: 'We highly recommend that developers use the `.webp` extension when fetching emojis so they're rendered as WebP for maximum  performance and compatibility.' This makes it clear we're talking about the file extension used when retrieving existing emojis.